### ISSUE TITLE
I need help with this (Poco X6 5G)

### ISSUE THREAD

**Issue by PeppiPlayz** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#176]
I've already moved the php folder into my C: drive, but I keep getting this error. Anyone willing to help?

![image](https://github.com/user-attachments/assets/62011c4c-8719-43a3-8ba2-63800f36d7a9)

**Comment by thaduongtbpro** [comment]
chụi kakka

**Comment by BlazingRiser** [comment]
Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue.

**Comment by vpolinov** [comment]
> settings apk ... should fix this issue.

WHO told you that? Why you repeat this crap?  FYI: settings app is no more updatable, thanks to xlaomi.

**Comment by BlazingRiser** [comment]
> > settings apk ... should fix this issue.
> 
> WHO told you that? Why you repeat this crap? FYI: settings app is no more updatable, thanks to xlaomi.

Yeah, my bad. Bypass is not supported for redmi note 13/poco x6 as listed in status.md.

It's updatable for devices listed in [status md](https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/blob/master/status.md) as supported, unsupported devices are unable to update or change settings app, as you have said thanks to xiaomi, additionally blocked devices will get error 30001(forced to verify device by Xiaomi).
